VideoAPI/VideoApi.IntegrationTests/Database/Commands/RemoveMessagesForConferenceCommandTests.cs

```diff
@@ -0,0 +1,65 @@
+using System;
+using System.Threading.Tasks;
+using FluentAssertions;
+using NUnit.Framework;
+using VideoApi.DAL;
+using VideoApi.DAL.Commands;
+using VideoApi.Domain;
+using System.Collections.Generic;
+using System.Linq;
+using VideoApi.DAL.Exceptions;
+using VideoApi.DAL.Queries;
+using Task = System.Threading.Tasks.Task;
+
+namespace VideoApi.IntegrationTests.Database.Commands
+{
+    public class RemoveMessagesForConferenceCommandTests : DatabaseTestsBase
+    {
+        private RemoveMessagesForConferenceCommandHandler _handler;
+        private GetConferenceByIdQueryHandler _conferenceByIdHandler;
+        private Guid _newConferenceId;
+
+        [SetUp]
+        public void Setup()
+        {
+            var context = new VideoApiDbContext(VideoBookingsDbContextOptions);
+            _handler = new RemoveMessagesForConferenceCommandHandler(context);
+            _conferenceByIdHandler = new GetConferenceByIdQueryHandler(context);
+            _newConferenceId = Guid.Empty;
+        }
+
+        [Test]
+        public void should_throw_conference_not_found_exception_when_conference_does_not_exist()
+        {
+            var conferenceId = Guid.NewGuid();
+            var command = new RemoveMessagesForConferenceCommand(conferenceId);
+            Assert.ThrowsAsync<ConferenceNotFoundException>(() => _handler.Handle(command));
+        }
+
+        [Test]
+        public async Task should_remove_messages_from_conference()
+        {
+            var seededConference = await TestDataManager.SeedConference();
+            TestContext.WriteLine($"New seeded conference id: {seededConference.Id}");
+            _newConferenceId = seededConference.Id;
+                       
+            var command = new RemoveMessagesForConferenceCommand(_newConferenceId);
+            await _handler.Handle(command);
+
+            var conference = await _conferenceByIdHandler.Handle(new GetConferenceByIdQuery(_newConferenceId));
```

Comment:
'GetConferenceByIdQuery' does not include messages. This will always return an empty list. please directly call the dbcontext and pull in the conference with messages to  confirm the list is empty

Reply:
Addressed it now